plugins/source/gcp/resources/services/networkconnectivity/locations.go

```diff
@@ -0,0 +1,52 @@
+package networkconnectivity
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v14/arrow"
+	"google.golang.org/api/networkconnectivity/v1"
+
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Locations() *schema.Table {
+	return &schema.Table{
+		Name:        "gcp_networkconnectivity_locations",
+		Description: `https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations/list`,
+		Resolver:    fetchLocations,
+		Multiplex:   client.ProjectMultiplexEnabledServices("networkconnectivity.googleapis.com"),
+		Transform:   client.TransformWithStruct(&networkconnectivity.Location{}, transformers.WithPrimaryKeys("Name")),
+		Columns: []schema.Column{
+			{
+				Name:       "project_id",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   client.ResolveProject,
+				PrimaryKey: true,
+			},
+		},
+		Relations: []*schema.Table{
+			InternalRanges(),
```

Comment:
We saw our share of bugs related to listing relation tables twice (as relations and in the top-level), so it's a good idea to make relation tables unexported.
```suggestion
			internalRanges(),
```

Reply:
done